### PR TITLE
Fix #4056 - rename DEPS to R2DEPS

### LINKED ISSUE #4056: crosscompile of latest release fails
```
arm-linux-musleabihf-gcc -L/home/rofl/x-prefix/armv7hf/lib -isystem /home/rofl/x-prefix/armv7hf/include -c -g -fdata-sections -ffunction-sections -Os -fno-unwind-tables -fno-asynchronous-unwind-tables -Wa,--noexecstack -MD   -fPIC -g -Wall -D__UNIX__=1 -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/../shlr/zip/include -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include -fPIC -Iinclude -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include -o src/packet.o src/packet.c
arm-linux-musleabihf-gcc -L/home/rofl/x-prefix/armv7hf/lib -isystem /home/rofl/x-prefix/armv7hf/include -c -g -fdata-sections -ffunction-sections -Os -fno-unwind-tables -fno-asynchronous-unwind-tables -Wa,--noexecstack -MD   -fPIC -g -Wall -D__UNIX__=1 -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/../shlr/zip/include -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include -fPIC -Iinclude -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include -o src/libgdbr.o src/libgdbr.c
arm-linux-musleabihf-gcc -L/home/rofl/x-prefix/armv7hf/lib -isystem /home/rofl/x-prefix/armv7hf/include -fPIC -shared -o lib/libgdbr.so -g -fdata-sections -ffunction-sections -Os -fno-unwind-tables -fno-asynchronous-unwind-tables -Wa,--noexecstack -MD   -fPIC -g -Wall -D__UNIX__=1 -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/../shlr/zip/include -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include -fPIC -Iinclude -I/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/include src/utils.o src/messages.o src/core.o src/packet.o src/libgdbr.o -Wl,--gc-sections -Wl,-z,relro,-z,now -Wl,-rpath-link=/home/rofl/x-prefix/armv7hf/lib -L/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/socket -lr_socket -L/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/util -lr_util -lutil  build build   -fPIC /home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/libr/../shlr/zip/librz.a
arm-linux-musleabihf-gcc: error: build: No such file or directory
arm-linux-musleabihf-gcc: error: build: No such file or directory
make[2]: *** [lib/libgdbr.a] Error 1
make[2]: Leaving directory `/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/shlr/gdb'
make[1]: *** [preall] Error 2
make[1]: Leaving directory `/home/rofl/x-prefix/armv7hf/src/build/radare2/radare2-0.10.0/shlr'
make: *** [all] Error 2
```

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**
Fixed the name clashes in some crosscompilation cases by renaming `DEPS` variable in the Makefiles to `R2DEPS`. Also cleaned up `sys/build.sh` a bit with the help of shellcheck.

...

**Test plan**

...

**Closing issues**

closes #4056 

...

### CODE REVIEW

File: sys/build.sh [hunk 1]
```diff
@@ -93,28 +93,28 @@ if [ "${USE_CS5}" = 1 ]; then
...
 	export HOST_CC=cc
 fi
 
 # purge first
 if [ "${PREFIX}" != "/usr" ]; then
-	A=`readlink /usr/bin/radare2 2>/dev/null`
+	A=$(readlink /usr/bin/radare2 2>/dev/null)
 	B="${PWD}/binr/radare2/radare2"
-	if [ -n "$A" -a ! -f "$A" ]; then
+	if [ -n "$A" ] && [ ! -f "$A" ]; then
```
Comment (radare) [inline sys/build.sh:117]: why this change? -a should be portable

Comment (XVilka) [reply sys/build.sh:117]: It is not - see https://github.com/koalaman/shellcheck/wiki/SC2166

Comment (wargio) [reply sys/build.sh:117]: i agree with the changes. `&&` is more portable.

File: sys/build.sh [hunk 2]
```diff
@@ -38,7 +38,7 @@ BuildJobsThrottler(){
 	echo "So, the build will run on $MAKE_JOBS job(s)."
 }
 
-if [ "`uname`" = Linux ]; then
+if [ "$(uname)" = Linux ]; then
```
Comment (wargio) [inline sys/build.sh:41]: i saw there are plenty of `uname` cmds. can you convert it to a local variable and use that in `if` and `elif` ?

File: sys/build.sh (hunk 1 above)
Comment (trufae) [reply sys/build.sh:117]: it's equally portable, the problem only arises when the string starts with a dash, but im fine with the change

**Comment by XVilka** [comment]
Regarding separating changes - it just can be merged without squashing.

**Pull request merged**